### PR TITLE
loosen requirements

### PR DESCRIPTION
**Pull request by dhpollack**
The requirements of this package are unnecessarily strict.  Specifically, I am having issues with tqdm. I have a more in-depth explaination in the issue that I create centre-for-humanities-computing/DaCy#75.  There are also a few optimizations to your setup.py file.  I notice that the requirements.txt file is not used, which could cause a mismatch when doing `pip install -r requirements.txt` and `pip install .`

### CODE REVIEW

**Comment by KennethEnevoldsen** [comment]
Hei @dhpollack sorry I missed this. I have debated the use of requirements.txt and setup being split up. I notice that other libraries keep these split up indicating that requirements also include packages for running the test suite. We, however, haven't done this so far so will change this according to your changes.

**Comment by dhpollack** [comment]
> Hei @dhpollack sorry I missed this. I have debated the use of requirements.txt and setup being split up. I notice that other libraries keep these split up indicating that requirements also include packages for running the test suite. We, however, haven't done this so far so will change this according to your changes.

I actually ran into an error when running this and just used your version.  But my issue was with importing `__version__` instead of the way you are doing it.  The issue was that loading the version also loaded `__init__.py` which loads a bunch of modules before they are installed.  The simplest solution would be to directly load those modules instead of putting them in the `__init__.py` but that does change the user-friendliness of the library a little bit.

File: setup.py [hunk 1]
```diff
@@ -1,16 +1,11 @@
...
-        if l.startswith("__version__"):
-            __version__ = l.split('"')[-2]
+from dacy.about import __version__
 
 with open("README.md", encoding="utf-8") as f:
     long_description = f.read()
 
 with open("requirements.txt", encoding="utf-8") as f:
-    requirements = f.read()
+    requirements = [l.strip() for l in f]
```
Comment (KennethEnevoldsen) [inline setup.py:8]:
```suggestion
    requirements = f.read().split("\n")
```

**Review by KennethEnevoldsen — CHANGES_REQUESTED** [review]

File: setup.py [hunk 2]
```diff
@@ -1,16 +1,11 @@
 import setuptools
-
-with open("dacy/about.py") as f:
-    v = f.read()
-    for l in v.split("\n"):
-        if l.startswith("__version__"):
-            __version__ = l.split('"')[-2]
+from dacy.about import __version__
```
Comment (KennethEnevoldsen) [inline setup.py:2]:
```suggestion
with open("dacy/about.py") as f:
     v = f.read()
     for l in v.split("\n"):
         if l.startswith("__version__"):
             __version__ = l.split('"')[-2]
```

File: setup.py [hunk 3]
```diff
@@ -1,16 +1,15 @@
...
-
 with open("dacy/about.py") as f:
-    v = f.read()
-    for l in v.split("\n"):
-        if l.startswith("__version__"):
-            __version__ = l.split('"')[-2]
+     v = f.read()
+     for l in v.split("\n"):
+         if l.startswith("__version__"):
+             __version__ = l.split('"')[-2]
```
Comment (KennethEnevoldsen) [inline setup.py:6]:
```suggestion
    v = f.read()
    for l in v.split("\n"):
        if l.startswith("__version__"):
            __version__ = l.split('"')[-2]
```

**Review by KennethEnevoldsen — COMMENTED** [review]
fixed indentation

**Comment by KennethEnevoldsen** [comment]
Sorry for the late response on this @dhpollack, seems like there has been an update in one of the workflow packages (nothing wrong with DaCy). I have just updated the workflow scripts in #80 will merge this as soon as that one is done. Enjoy the holidays.

**Comment by KennethEnevoldsen** [comment]
@dhpollack I have merged these changes in #82 into the dev branch along with a few other PRs related to GitHub workflow to merge a conflict before committing it to the main branch.

Thanks again for the addition sorry for the late results. Wanted to do an overhaul of the Github actions which ended up taking longer than expected.

Enjoy the holidays 🎄

**Pull request closed without merging**